app/queries/user_standings_resolver.rb

```diff
@@ -0,0 +1,17 @@
+class UserStandingsResolver < ApplicationQuery
+  include AuthorizeSchoolAdmin
+  property :student_id
```

Comment:
The query says `UserStanding` and you are taking `Student Id` as input; I would expect the input to be `user_id` instead of `student_id`

Reply:
> The query says UserStanding and you are taking Student Id as input; I would expect the input to be user_id instead of student_id

The user standings (Standing logs) are linked to user as it is a school level thing.

Currently, the user_standings are displayed on the school student page (/school/students/:id/standing) which has only the student_id, for getting the user_id we need to make another request to the backend, which I think it is not necessary.

> Currently, the user_standings are displayed on the school student page (/school/students/:id)

Currently, we don't have a separate page for users similar to coaches(/school/coaches)

Reply:
In that case, i would rename the resolver to `StudentStandingsResolver`

Reply:
@bodhish 

Fixed in this https://github.com/pupilfirst/pupilfirst/pull/1468/commits/dae143aeebe39e072104aacea33bc710ad22ec7f

After glancing at the code for some time, it struck that I can pull the `Id`  of the user in the initial graph query, which is already getting user data